packages/experimental/src/components/AboutModal/AboutModal.js

```diff
@@ -0,0 +1,212 @@
+/**
+ * Copyright IBM Corp. 2020, 2020
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  ComposedModal,
+  Link,
+  ModalHeader,
+  ModalFooter,
+  ModalBody,
+  Tabs,
+  Tab,
+} from 'carbon-components-react';
+
+import { expPrefix } from '../../global/js/settings';
+
+export const AboutModal = ({
+  copyrightText,
+  enableScroll,
+  legalText,
+  links,
+  logo,
+  productName,
+  versionNumber,
+  onRequestClose,
+  body,
+  open,
+  technologiesUsed,
+  theme,
+}) => {
+  return (
+    <ComposedModal
+      className={[
+        `${expPrefix}-about-modal`,
+        theme === 'dark'
+          ? `${expPrefix}-about-modal-dark-theme`
+          : `${expPrefix}-about-modal-light-theme`,
+        enableScroll ? `${expPrefix}-about-modal-scroll-enabled` : '',
```

Comment:
Lose this scroll enabled class

Reply:
This is now handled by the `react-resize-detector` component and adds a class based on what that component tells us about the modal height vs the modal content height